File: sample-rest-api/src/main/java/nl/haploid/octowight/sample/configuration/AppConfiguration.java
package nl.haploid.octowight.sample.configuration;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;

@Configuration
public class AppConfiguration {

	@Bean
	public static PropertySourcesPlaceholderConfigurer propertyPlaceholderConfigurer() {
		return new PropertySourcesPlaceholderConfigurer();
	}
}

File: sample-rest-api/src/main/scala/nl/haploid/octowight/sample/configuration/AppConfiguration.scala
package nl.haploid.octowight.sample.configuration

import org.springframework.context.annotation.{Configuration, PropertySource, Bean}
import org.springframework.context.support.PropertySourcesPlaceholderConfigurer

object AppConfiguration {

  @Bean def propertyPlaceholderConfigurer = new PropertySourcesPlaceholderConfigurer
}

@PropertySource(value = Array("file:./local.properties"), ignoreResourceNotFound = true)
@Configuration
class AppConfiguration
